### PR TITLE
Fix MSP Radio emulator in PWA

### PR DESCRIPTION
**Pull request by McGiverGim**
This makes working again the MSP radio emulator under PWA.
With this changes, it works under PWA and under NW.js.

image

BUT I've marked it as draft because under PWA only works with `yarn dev` but not under `yarn review`. I suspect it has something to do with https://vitejs.dev/guide/build#multi-page-app but I'm not sure. I tried to add this config to the code without luck. Some idea?

### CODE REVIEW

**Comment by McGiverGim** [comment]
Added a new commit that fixes the issue with the "build" version of PWA. Now works, but maybe my solution is not good enough, but we can make it better with the reviews.

File: src/receiver_msp/index.html
Comment (chmelevskij) [inline src/receiver_msp/index.html:1]: there is not need really to rename this if we specify the input with `rollupOptions`

**Review by chmelevskij — APPROVED** [review]

File: src/receiver_msp/index.html
Comment (haslinghuis) [reply src/receiver_msp/index.html:1]: Not really a tab - could we rename it to `/receiver_msp.html` instead?

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by McGiverGim** [comment]
I tried but it didn't work. My first idea was to add it as subfolder inside the tabs one, and without renaming it, but it didn't work. Maybe I did something wrong, but I didn't found the problem.

File: src/receiver_msp/index.html
Comment (McGiverGim) [reply src/receiver_msp/index.html:1]: I tried but it didn't work. My first idea was to add it as subfolder inside the tabs one, and without renaming it, but it didn't work. Maybe I did something wrong, but I didn't found the problem.

**Review by haslinghuis — APPROVED** [review]
Possible have something to do with the `rollupOptions` used in Vite

File: src/receiver_msp/index.html
Comment (McGiverGim) [reply src/receiver_msp/index.html:1]: @haslinghuis it seems now it works, I don't know why yesterday don't. I will push a new PR renaming it.

**Pull request merged**